google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -47,8 +47,9 @@ appengine.cloudsdk.java.components.howtoinstall=To install, run "gcloud componen
 appengine.cloudsdk.location.label=Cloud SDK directory\:
 appengine.cloudsdk.location.browse.window.title=Browse to Cloud SDK directory
 appengine.cloudsdk.location.badchars.message=The specified Cloud SDK directory contains invalid characters or starts or ends with spaces.
-appengine.cloudsdk.location.invalid.message=No Cloud SDK was found in this directory.
+appengine.cloudsdk.location.invalid.message=No Cloud SDK was found in the specified directory.
 appengine.cloudsdk.location.missing.message=Cloud SDK home directory is not specified.
+appengine.cloudsdk.location.missing.message.deployui=The Cloud SDK is misconfigured. To fix, reconfigure the Server above.
```

Comment:
I thought it would make it more explicit when to use. The message is too specific, it doesn't make sense to use it anywhere else other than in the UI. Maybe whoever wants to use it can see that from the message itself, though?

Regarding misconfigured, I wanted that footer UI message to be in line with the other Misconfigured messages in the Devappserver UI. There is an issue in the message name -- appengine.cloudsdk.location.missing.message.deploy, s/missing/invalid